spec/lib/js_fetcher_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+describe JsFetcher do
+  # Below live site will pinged on each test run, should be handled with VCR cassette config.
```

Comment:
@krbhavith As mentioned in standup, please update this comment to add the JIRA ticket number for the follow-on ticket you have created/will create for resolving the vcr configuration issues at some future date.  This saves this comment from getting lost in the code.

Reply:
Updated comments with the Jira ticket number